src/context.rs

```diff
@@ -25,7 +25,7 @@ impl ProverContext {
         let device_alloc = StaticDeviceAllocator::init_all(block_size)?;
 
         let small_host_alloc = SmallStaticHostAllocator::init()?;
-        let host_alloc = StaticHostAllocator::init(block_size, 1 << 8)?;
```

Comment:
Were instances of this just a bug? I notice `StaticHostAllocator::init`'s arguments are `(num_blocks: usize, block_size: usize)` _before_ the PR's diffs (and the PR doesn't change the signature).

Reply:
yes, the arguments were swapped but those classes are not used anywhere yet so it had no effect.